### DISCUSSION TITLE
Create community lead Jquery Mobile repository

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [jquery/jquery#5092] [Ideas]
Hi @mgol friendly ping
Jquery mobile is no longer maintained by the jquery organization https://blog.jquerymobile.com/2021/10/07/jquery-maintainers-continue-modernization-initiative-with-deprecation-of-jquery-mobile/

HOWEVER the current repository is frozen.. 
https://github.com/jquery-archive/jquery-mobile
The truth is: jquery-mobile has received 13000 commits and many have been made by DOM/CSS experts, to a level of correctness, accessibility, simplicity and optimization that is mostly unmatched in alternative UI libraries.

The only thing I'm asking is that you create a new repository, that would be the official jquery-mobile-community-maintained
On this fork the community could still merge fixes and improvements.
https://github.com/jquery-archive/jquery-mobile would stay as the last officially reviewed code.
Without an officially recognized fork, the community doesn't know where to contribute and therefore does not contribute.

Please make this happen, it would be unfortunate for those key useful and battletested components to become totally unmaintained.

**Comment by mgol** [comment]
It’s not that simple. There’s a certain level of trust associated with the jQuery name. Someone would need to vet candidates, help them with issues related to the project setup, maintain infrastructure, do releases, receive security issues.

We barely have resources for basic maintenance of jQuery UI, we have none for Mobile.

If you find an interested group, the code is open source - feel free to fork it and continue development under a different name. We won’t be involved in that, though.
